### PR TITLE
Feature/(blo 22) refactor structure

### PR DESCRIPTION
**Pull request by hugovandevliert**
Restructured application structure.
- Moved application logic to `src` folder

### CODE REVIEW

**Review by saltz — APPROVED** [review]

File: src/lib/p2p/initialconnector.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = class InitialConnector {
 
   constructor(firstTimeout) {
     if (!instance) {
-      this.ip = 'http://p2p:8082';
+      this.ip = 'http://145.93.57.202:8082';
```
Comment (quintaartsen) [inline src/lib/p2p/initialconnector.js]: revert this change

**Review by quintaartsen — APPROVED** [review]

**Review by saltz — APPROVED** [review]
